packages/gatsby/content/getting-started/install.md

```diff
@@ -10,7 +10,7 @@ order: 2
 
 The preferred way to manage Yarn is through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html), a new binary shipped with all Node.js releases starting from 16.10. It acts as an intermediary between you and Yarn, and lets you use different package manager versions across multiple projects without having to check-in the Yarn binary anymore.
 
-### Node.js >=16.10
+### Node.js ^16.10
```

Comment:
```suggestion
### Node.js >=16.10
```
`^16.10` is the same as `>=16.10 <17` which isn't correct, Corepack is present in all releases after `16.10.0`